source/connect/connection-options.txt

```diff
@@ -0,0 +1,286 @@
+.. _cpp-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+Overview
+--------
+
+This page describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection by using
+options in the connection URI.
+
+.. note::
+
+   You can set most connection options by using either the connection URI or the
+   ``client_options`` parameter. However, some options support only one of these
+   methods. You might need to use both a connection URI and the ``client_options``
+   parameter to set all the options that you need.
+
+.. _cpp-connection-uri:
+
+Using the Connection URI
+------------------------
+
+When you construct a ``mongocxx::client`` object, you can pass in a ``mongocxx::uri``
+object that represents your connection URI. This connection URI can include
+connection options as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``tls`` option with a value of ``true`` and the
+``tlsCertificateKeyFile`` option with a value of ``path/to/file.pem``:
+
+.. literalinclude:: /includes/connect/connection-options.cpp
+   :language: cpp
+   :copyable: true
+
+Connection URI Options
+----------------------
+
+The following sections describe the connection options that you can set in the connection
+URI passed to the {+driver-short+}. Each connection option links to its entry in
+the {+mdb-server+} manual. If you can access the value of a connection option by using
+a member of the ``mongocxx::uri`` class, the member name is also listed. 
+
+Replica Set Options
+~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Connection URI Option
+     - ``mongocxx::uri`` Member
+
+   * - :manual:`directConnection </reference/connection-string/#mongodb-urioption-urioption.directConnection>`
+     - `direct_connection() <https://mongocxx.org/api/current/classmongocxx_1_1v__noabi_1_1uri.html#a4d4e43c3ef3f30f81c990c04082b1655>`__
+   
+   * - :manual:`replicaSet </reference/connection-string/#mongodb-urioption-urioption.replicaSet>`
+     - `replica_set() <https://mongocxx.org/api/current/classmongocxx_1_1v__noabi_1_1uri.html#ad2d71ae778881db9ffc64fc7c07e0dad>`__
+   
+Connection Options
+~~~~~~~~~~~~~~~~~~
+
+TLS Options
+```````````
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Connection URI Option
+     - ``mongocxx::uri`` Member
+
```

Comment:
S: I think it makes sense to add a "mongocxx::client::options Member" row - especially to this section's table, since seems like most of the options you can set in an options parameter are tls / ssl related (from [API docs](https://mongocxx.org/api/current/classmongocxx_1_1v__noabi_1_1options_1_1client.html)).
```suggestion
- ``mongocxx::client::options`` Member
```

Reply:
LMK if other page changes are sufficient here.